test/metabase/query_processor/expand_resolve_test.clj

```diff
@@ -153,12 +177,18 @@
     :type     :query
     :query    {:source-table (id :checkins)
                :filter       {:filter-type :>
-                              :field       {:field-id      (id :users :last_login)
-                                            :fk-field-id   (id :checkins :user_id)
-                                            :datetime-unit :year}
-                              :value       {:field-placeholder {:field-id      (id :users :last_login)
-                                                                :fk-field-id   (id :checkins :user_id)
-                                                                :datetime-unit :year}
+                              :field       {:field-id           (id :users :last_login)
+                                            :fk-field-id        (id :checkins :user_id)
+                                            :datetime-unit      :year
+                                            :remapped-from      nil
+                                            :remapped-to        nil
+                                            :field-display-name nil}
+                              :value       {:field-placeholder {:field-id           (id :users :last_login)
```

Comment:
Instead of having to update a million fields it's probably better to do something like 

```clojure
(def ^:private field-placeholder-defaults
  {:remapped-from      nil
   :remapped-to        nil
   :field-display-name nil})

...

{:field-placeholder (merge field-placeholder-defaults 
                           {...}}
```

that way when new things are added to the fields in the future you only need to update one thing instead of dozens of tests.

(We are following a similar pattern in other places in the code -- search for `card-defaults` for example)